docs/release/3.4/release-3-4-12.md

```diff
@@ -0,0 +1,143 @@
+OSG Software Release 3.4.12
+===========================
+
+**Release Date**: 2018-05-10
+
+Summary of changes
+------------------
+
+!!! note
+The repositories referenced by osg-release will be retired on May 15th. Update osg-release before that date.
```

Comment:
I don't think this note will format correctly: the text in the note needs to be preceded by 4 spaces. Same goes for the same note in OSG 3.3

Reply:
Let's actually change `!!! note` to `!!! warning "Required Action"`